src/components/Filter/FilterInput.vue

```diff
@@ -395,12 +400,14 @@ export default {
         this.$emit('focus-prev');
       }
     },
+    handleFocus() {
+      this.showSuggestedTags = true;
+    },
   },
   created() {
     if (
       this.focusInputWhenCreated
       && document.activeElement !== this.$refs.input
-      && this.inputIsNotEmpty
```

Comment:
This change potentially breaks other use cases for users who want the input to focus only when it's not empty.
Could you create another prop that make sure that we focus input when it's empty too?
Something like `focusInputWhenEmpty` and then the condition would be:

`this.inputIsNotEmpty && !focusInputWhenEmpty`

Thinking out loud. You may find a better way to do it :)

Reply:
[54a559e](https://github.com/apple/swift-docc-render/pull/396/commits/54a559ef81ba62970191dfba7f9be5fd92e3a6a6)